edx_rest_framework_extensions/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '8.1.0'  # pragma: no cover
+__version__ = '8.1.1'  # pragma: no cover
```

Comment:
Please make this 8.2.0 (minor bump in version).

Reply:
bumped to 8.3.0 since 8.2.0 was found on master after rebasing.